main.nf

```diff
@@ -313,61 +313,60 @@ process alignWithBowtie {
     """
 }
 
+
 /*
- * Sort the bam file with samtools and index the result.
+ * Take the reads and process them to get the remapped variants
+ *
  */
-process sortBam {
+process readsToRemappedVariants {
 
     input:
         path "reads_aligned.bam" from reads_aligned_bam
+        path "genome.fa" from params.newgenome
 
     output:
-        path "reads_aligned.sorted.bam" into reads_sorted_bam
-        path "reads_aligned.sorted.bam.bai" into bam_index
+        path "variants_remapped.vcf" into variants_remapped
 
     """
-    samtools sort reads_aligned.bam -o reads_aligned.sorted.bam
-    samtools index reads_aligned.sorted.bam
+    # Ensure that we will use the reads_to_remapped_variants.py from this repo
+    ${baseDir}/variant_remapping_tools/reads_to_remapped_variants.py -i reads_aligned.bam \
+        -o variants_remapped.vcf -f $params.flankingseq \
+        -s $params.scorecutoff -d $params.diffcutoff --max_alignment 1 \
```

Comment:
I've read the notes in the JIRA ticket; the precision of 0.999994 is beyond impressive, but the recall slipping to 0.8769 is bothering me a bit.

For the time being, I think it's a very good solution, but I'm sure in the future, if we add more metrics and more sophisticated ways of combining them, we could rescue some of those multiple-alignment variants as well.

Reply:
(Increased flanking sequence length could also help, I imagine.)

Reply:
Yes that is completely correct. 
In my test so far with short flanking region it is impossible to know which is the correct alignment when multiple alignment exists. In some cases, the best matching alignment is not the correct one. 
The idea behind this new filter is to ensure that **ALL** variants remapped are correct and try again with longer flanking regions for the ones failing there filter.